praw/models/reddit/submission.py

```diff
@@ -441,6 +441,26 @@ class SubmissionModeration(ThingModerationMixin):
 
     REMOVAL_MESSAGE_API = "removal_link_message"
 
+    @cachedproperty
+    def flair(self):
+        """Provide an instance of :class:`.SubmissionModerationFlair`.
+
+        Allows the moderator to work with setting flairs on a submission
+            that use the template system introduced in the redesign.
+
+        If a flair template is not being used, then a moderator should call
+            :meth:`.set_flair` to set a flair without a template.
+
+        Example usage:
+
+        .. code:: python
+
+            choices = submission.mod.flair.choices()
+            template_id = choices[0]["flair_template_id"]
+            submission.mod.flair.select(template_id, text="Custom flair")
+
+        """
```

Comment:
Does this do anything?

Reply:
It sets a flair using the first available template. However, it will fail if there are no templates.